flink-sql-runner/src/main/java/com/github/streamshub/flink/SqlRunner.java

```diff
@@ -36,25 +36,35 @@ public class SqlRunner {
     private static final String STATEMENT_DELIMITER = ";"; // a statement should end with `;`
     private static final String LINE_DELIMITER = "\n";
 
+    private static final Pattern STATEMENT_DELIMETER_PATTERN = Pattern.compile("(?<!\\\\)" + STATEMENT_DELIMITER);
+
     private static final Pattern SET_STATEMENT_PATTERN =
-            Pattern.compile("SET\\s+'(\\S+)'\\s+=\\s+'(.*)';", Pattern.CASE_INSENSITIVE);
+            Pattern.compile("SET\\s+'(\\S+)'\\s*=\\s*'(.*)'\\s*;", Pattern.CASE_INSENSITIVE);
+
+    private static final Pattern STATEMENT_SET_START_PATTERN =
+            Pattern.compile("\\s?EXECUTE\\s+STATEMENT\\s+SET\\s+BEGIN\\s+", Pattern.CASE_INSENSITIVE);
 
-    private static final Pattern STATEMENT_SET_PATTERN =
-            Pattern.compile("(EXECUTE STATEMENT SET BEGIN.*?END;)", Pattern.CASE_INSENSITIVE | Pattern.DOTALL);
+    private static final Pattern STATEMENT_SET_END_PATTERN = Pattern.compile("\\s*END\\s*;", Pattern.CASE_INSENSITIVE);
 
     public static void main(String[] args) throws Exception {
-        if (args.length != 1) {
-            throw new Exception("Exactly 1 argument is expected.");
+
+        System.out.println("\n####################################\n");
+        System.out.println("THESE ARE THE ARGS I HAVE BEEN SENT:");
```

Comment:
Logger at debug...

Reply:
Well that definitely shouldn't still be in there. You are seeing the edges of my madness, trying to figure out what K8s was actually passing to the main method...